changelog.md

```diff
@@ -32,6 +101,8 @@ You can access Lighthouse Treemap from the report:
 
 Currently, only reports generated with the Lighthouse Node CLI will connect to the Lighthouse Treemap App. We are working on adding this functionality to Lighthouse in DevTools and PageSpeed Insights.
 
+[Demo](https://googlechrome.github.io/lighthouse/treemap/?gist=30a18304de56e7be08f10976a1b11702)
```

Comment:
the treemap section alone may soon be bigger than any individual version's release log (except 7.5.0 itself, of course) :P

More seriously, this is going to rapidly get pushed down the changelog. Should we expand `lighthouse-treemap/readme.md` instead?

Reply:
(fyi I added this link to the GH changelog post-release as an after thought. it was to the viewer. I just updated it to link to treemap directly, and added that link to the changelog file here)

All the wording here is a point-in-time commentary on the feature, and I don't see a reason to be concerned about the length of releases or sections here, so I don't think it matters.

Reply:
> All the wording here is a point-in-time commentary on the feature, and I don't see a reason to be concerned about the length of releases or sections here, so I don't think it matters.

I mean it's a reference for changes, not a blog post. IMO even major changes should only have enough description to give an idea why they're significant, with an issue/PR link if someone wants to learn more. Otherwise it sabotages using the file as a cross-reference/development timeline. But that's just the classic release notes/changelog discussion...

> Or do you just mean also adding the link to that other readme file?

Yes, this is what I meant in the actually constructive part of my comment. This is going to be pushed down and in a release or two (relatively) no one is ever going to see it. We should add this to the treemap/readme.md (or wherever) so it's still in a place people will look for it.